solution/static-assets/requirements.txt

```diff
@@ -8,7 +8,6 @@ requests
 djangorestframework
 psycopg2-binary
 django-solo
-django-opensearch-dsl
```

Comment:
we dont use this. A long time ago we thought about using it and its just been sitting in our requirements.  Our search is all in Postgres FTS.